src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar-month-length">
```

Comment:
The class `calendar-month-length` is not descriptive enough. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), classes should be named clearly and unambiguously. Consider specifying the month length in the class name, like `calendar--month-length-31`.